src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+    <p className="Person__partner">
+      {person.isMarried === true &&
+        person.sex === 'm' &&
+        `${person.partnerName} is my wife`}
+      {person.isMarried === true &&
+        person.sex === 'f' &&
+        `${person.partnerName} is my husband`}
+      {person.isMarried === false && `I am not married`}
```

Comment:
too many conditionals here. try to simplify this expressions.
you can create variable for instance person partner

```javascript
 const partner = sex === 'm' ? 'wife' : 'husband';
``` 

and then use conditional rendering 

```javascript
        { isMarried
          ? `${partnerName} is my ${partner}`
          : 'I am not married'
        }
``` 

and you don't need to compare isMarried with true since it is boolean type